client/sections-preload.js

```diff
@@ -25,11 +25,11 @@ export const hub = {};
 emitter( hub );
 
 /**
- * Preload a section by name
+ * Preload a section by name.
  *
  * @export
- * @param {String} section The named section to load
+ * @param {String} sectionName The name of the section to load
  */
-export function preload( section ) {
-	hub.emit( 'preload', section );
+export function preload( sectionName ) {
+	hub.emit( 'preload', sectionName );
```

Comment:
Is the rename here to clarify that the sectionName is a string?

Reply:
Yes, exactly. That's part of [this change](https://github.com/Automattic/wp-calypso/pull/19396/commits/cee500a1bd097b51d1965046765debec9a9714c3).